Makefile

```diff
@@ -10,12 +10,20 @@ cidocker = docker run ${ciargs} ${dockerargs} golang:1.4
 all: ${build_image}.created build check integration-tests
 
 integration-tests:
+	@if [ ! -e ./cmd/dnet/dnet ]; then \
```

Comment:
You don't need to write shell scripts for a target binary which is a dependency of `integration-tests`. All you need is the following:
```
integration-tests: ./cmd/dnet/dnet
...
...

./cmd/dnet/dnet:
        godep go build -o $@ ./cmd/dnet
```